code-examples/quarkus-kafka-quickstart/README.md

```diff
@@ -0,0 +1,32 @@
+Quarkus Kafka Quickstart
+========================
+
+Based on https://github.com/quarkusio/quarkus-quickstarts/blob/main/kafka-quickstart/README.md
+
+This project illustrates how you can interact with Apache Kafka using MicroProfile Reactive Messaging.
+
+## RHOAS addons
```

Comment:
Everything below here should be in the quick start. We don't need it hanging out in this random other file.